extras/systemd/heketi.env

```diff
@@ -0,0 +1,70 @@
+#
+# These environment variables will be loaded by systemd in case heketi is
+# started as a service (as opposed to running in a container).
+#
+# See the actual /etc/heketi/heketi.json configuration file for more options.
+#
+# The variables commented out show the default values. Variables not set here
+# might be set in the /etc/heketi/heketi.json configuration file.
+#
+
+# Database file name
+#HEKETI_DB_PATH=/var/lib/heketi/heketi.db
+
+# fstab file on nodes
+#HEKETI_FSTAB=/etc/fstab
+
+# Set log level. Choices are:
+#   none, critical, error, warning, info, debug
+#HEKETI_GLUSTERAPP_LOGLEVEL=warning
+
+# Heketi Server Port Number
+#HEKETI_HTTP_PORT=8080
+
+# Allow starting heketi when there are stale pending ops being in the db.
+#HEKETI_IGNORE_STALE_OPERATIONS=0
+
+# The maximum limit of snapshots. If no limit has been specified, then Heketi
+# will not set a limit and GlusterFS will use its default value.
+#HEKETI_SNAPSHOT_LIMIT=
+
+# Volume options that will be applied for all volumes created. Can be
+# overridden by volume options in volume create request.
+#HEKETI_PRE_REQUEST_VOLUME_OPTIONS=""
+
+# Volume options that will be applied for all volumes created. To be used to
+# override volume options in volume create request.
+#HEKETI_POST_REQUEST_VOLUME_OPTIONS=""
+
+# Creates Block Hosting volumes automatically if not found or exsisting volume
+# exhausted.
+#HEKETI_AUTO_CREATE_BLOCK_HOSTING_VOLUME=1
+
+# New block hosting volume will be created in size mentioned, This is
+# considered only if auto-create is enabled.
+#HEKETI_BLOCK_HOSTING_VOLUME_SIZE=500
+
+# New block hosting volume will be created with the following set of options.
+# Removing the group gluster-block option is NOT recommended. Additional
+# options can be added next to it separated by a comma.
+#HEKETI_BLOCK_HOSTING_VOLUME_OPTIONS="group gluster-block"
+
+# Execute plugin. Possible choices:
+# mock: This setting is used for testing and development. It will not send
+#       commands to any node.
+# ssh:  This setting will notify Heketi to ssh to the nodes. It will need the
+#       values in sshexec to be configured.
+# kubernetes: Communicate with GlusterFS containers over Kubernetes exec api.
+#HEKETI_EXECUTOR=mock
+
+# SSH username and private key file information
+#HEKETI_SSH_KEYFILE="path/to/private_key"
+#HEKETI_SSH_PORT=22
+#HEKETI_SSH_USER=sshuser
+
+# Rebalance on expansion is done by default, but it can be disabled if needed.
+#HEKETI_GLUSTERAPP_REBALANCE_ON_EXPANSION=1
+
+# The node health monitor is enabled by default. It can be disabled for
+# debugging.
+#HEKETI_DISABLE_HEALTH_MONITOR=0
```

Comment:
Lets drop this. User's shouldn't really be messing with this as its mostly for running heketi under test/deep debugging.

Reply:
Ok, will do.